src/models/combination.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose')
```

Comment:
Forgot small note.
This could be moved into JSON, no need to have is as a model.
@Mang1x